content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,136 @@
+---
+reviewers:
+- verb
+- yujuhong
+title: Debug a Running Pod
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+This page shows how to investigate problems in a running pod using a
+debugging ephemeral container. This is useful when the container being
+debugged does not include debugging tools.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* You should be familiar with the basics of
+  [Ephemeral Containers](/docs/concepts/workloads/pods/ephemeral-containers/).
+* The examples in this section require the `EphemeralContainers` [Feature
+  Gate](/docs/reference/command-line-tools-reference/feature-gates.md) to be
+  enabled and kubernetes client and server version of at least 1.16.
+* Install [kubectl debug](https://github.com/verb/kubectl-debug) using the
+  provided instructions.
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Creating a sample application
+
+First, create a sample application to be used with the rest of the steps
+in this task. This example uses a naked pod for simplicity.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sample-app
+spec:
+  shareProcessNamespace: true
+  containers:
+    - name: nginx
+      image: nginx
+```
+
+Create the pod described above in your cluster:
+
+```shell
+kubectl create -f sample-app.yaml
+```
+
+## Create an ephemeral container for debugging
+
+Using `kubectl debug`, create an ephemeral container for debugging:
+
```

Comment:
I thought I read in the concepts section that `kubectl edit` could not be used to manage the EC and that a `special` API handler is required. Can you explain how `kubectl debug` is used here?

Reply:
hello @verb. In general, looking good. I reread the task page. I did not validate the configuration code and commands.
- The concepts page example takes a different approach. Has the PR received a tech review?
- In lines 27-28, there is a link to debug instructions from your github account. I do not _think_ this is permitted. Would you check 
`the [Kubernetes Content Guide](/docs/contribute/style/content-guide/)`. Thanks!
```
enabled and kubernetes client and server version of at least 1.16.
* Install [kubectl debug](https://github.com/verb/kubectl-debug) using the
```

Reply:
@verb if you wanted to propose a blog post about `kubectl debug` and ephemeral containers, I think that'd be a nice way to share this info.

Reply:
👍

Reply:
> hello @verb. In general, looking good. I reread the task page. I did not validate the configuration code and commands.
> 
> * The concepts page example takes a different approach. Has the PR received a tech review?

No, who gives the tech review? Anyone I choose?

> * In lines 27-28, there is a link to debug instructions from your github account. I do not _think_ this is permitted. Would you check
>   `the [Kubernetes Content Guide](/docs/contribute/style/content-guide/)`. Thanks!

I asked in #sig-docs prior to opening this PR and they suggested linking to the unofficial repo. That being said, I'm not comfortable with this, either. I'm working on moving the plugin to a kubernetes-sigs repo, but I'm not sure it will be done by docs deadline.  When is docs deadline, btw?

cc @dims @pwittrock

Reply:
@simplytunde, can you comment on the docs deadline for the 1.16 release?

Reply:
Is there a plugin contrib area in the kubectl repo?

Reply:
There's no plugin contrib area in kubectl afaik. There's only the https://krew.dev index of where plugins live. (cc @ahmetb )

The deadline for submitting 1.16 docs is Monday, so we're out of time. I recommend that I split this PR in two and we merge the concept doc into dev-1.16 and the task into master at a later point when kubectl-debug moves into a kubernetes-owned repo. @simplytunde is that doable?

@kbhawkey I didn't know that I needed to bring my own technical reviewer. I can try to find one to do a last minute review, but I'm not sure anyone will be available. How would you like to proceed?

Reply:
@verb. You do not have to "bring your own technical reviewer" 😄 . I like your idea of holding off on the task page.

Reply:
we can get the task page into master after release.